src/Asset/Asset.php

```diff
@@ -394,4 +403,16 @@ private function prepareDefinitionInput(array $input): array
 
         return $input;
     }
+
+    public function getCloneRelations(): array
```

Comment:
IMHO, each capacity should declare its own `getCloneRelations()` method. Indeed, this is the only solution to permit to a capacity provided by a plugin to declare a new clone relation (even more if the class itself is a plugin class).